apps/opencs/model/world/defaultgmsts.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef CSM_WORLD_DEFAULTGMSTS_H
+#define CSM_WORLD_DEFAULTGMSTS_H
+
+#include <cstddef>
+
+namespace CSMWorld {
+    class DefaultGMSTs {
```

Comment:
Having a class with only static member functions is a bit pointless. Normally you would use a namespace in this case.